test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,17 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() {
+        // empty list
+        assertHasNoNull();
+
+        // list has null objects
+        assertHasNull(null, 1, 2, 3);
+
+        // list has no null objects
+        assertHasNoNull(1, 2, 3, 4);
```

Comment:
Good job on attempting this LO!

It's good that you separated your tests into different categories and tested for both True and False cases. As mentioned in tutorial, it might also be a good idea to test a mixture of different data types.